### PR TITLE
carousel modal fix

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: layouts/partials/carousel/host-mobile.css [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "right: 0;" +
     "bottom: 0;" +
     "left: 0;" +
-    "z-index: 1040;" +
+    "z-index: 110000;" +
```
Comment (sdorleTVPage) [inline layouts/partials/carousel/host-mobile.css]: Why do we need to increase the z-index ?

Comment (karloscastaneda) [reply layouts/partials/carousel/host-mobile.css]: just was following recommendations from https://tvpage.atlassian.net/browse/DEV-8743

**Comment by karloscastaneda** [comment]
@sdorleTVPage copy, will apply for all modal containig widgets.

File: layouts/partials/carousel/host-mobile.css (hunk 1 above)
Comment (sdorleTVPage) [reply layouts/partials/carousel/host-mobile.css]: @karloscastaneda : I don't think we need to increase the z-index.

**Pull request closed without merging**